.github/workflows/stable-snapshot-timestamp.yml

```diff
@@ -39,11 +39,10 @@ jobs:
       issues: 'write'
       pull-requests: 'write'
       contents: 'write'
-    steps:
-      - uses: sigstore/root-signing/.github/workflows/reuseable-snapshot-timestamp.yml@main
-        with:
-          snapshot_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot'
-          timestamp_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp'
-          repo: 'repository/'
-          provider: projects/237800849078/locations/global/workloadIdentityPools/root-signing-pool/providers/sigstore-root'
-          service_account: 'sigstore-root-signing@project-rekor.iam.gserviceaccount.com'
+    uses: sigstore/root-signing/.github/workflows/reuseable-snapshot-timestamp.yml@main
```

Comment:
missing the steps and the array

```yaml
name: Stable Snapshot and Timestamp

permissions: read-all

# Execute this as a biweekly cron job and on changes to repository/
# when new published metadata is submitted.
on:
  # Enable cron for re-signing snapshot and timestamp every week
  schedule:
    - cron: '0 0 */7 * *' # every 7 days
  # When a new root is staged
  push:
    branches:
      - main
    paths:
      - 'repository/staged/root.json'
  workflow_dispatch:

jobs:
  run_snapshot_timestamp_publish:
    runs-on: ubuntu-20.04
    permissions:
      id-token: 'write'
      issues: 'write'
      pull-requests: 'write'
      contents: 'write'
    steps:
    - uses: sigstore/root-signing/.github/workflows/reuseable-snapshot-timestamp.yml@main
      with:
        snapshot_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/snapshot'
        timestamp_key: 'gcpkms://projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp'
        repo: 'repository/'
        provider: projects/237800849078/locations/global/workloadIdentityPools/root-signing-pool/providers/sigstore-root'
        service_account: 'sigstore-root-signing@project-rekor.iam.gserviceaccount.com'
```

you can test the file here https://rhysd.github.io/actionlint/

Reply:
interesting, It seemed like reusable workflows couldn't be invoked in steps... let me see.
https://docs.github.com/en/actions/using-workflows/reusing-workflows

Reply:
ahh, this file is meant to be reusable in another workflow? if yes we need to rewrite that

Reply:
I think I see the issue: I can't use `runs-on` or `permissions`!

Reply:
and the `on`

Reply:
Hmm this is the caller workflow: it should have the `on`

Reply:
So looking at this example https://docs.github.com/en/actions/using-workflows/reusing-workflows#example-caller-workflow

(1) reusbale workflows are not called in steps, they're called as jobs.job_id.uses
(2) I cannot provide a `runs-on`, but I can give explicit perms